system/core/database.php

```diff
@@ -560,6 +560,44 @@ function serveFile($filename, $root) {
 			die("Invalid filename: $filename");
 		}
 		header('Content-Type: ' . getMimeType($filename));
+	  $filesize = filesize($filename);
+		$shortlen =$filesize-1;
+
+		if(getMimeType($filename)==='audio/mpeg'){
```

Comment:
No need to limit on mimetype, right? If the browser asks for a part of any file, it can just get it?

Reply:
Agreed :-)